protocols/luci-proto-qmi/htdocs/luci-static/resources/protocol/qmi.js

```diff
@@ -133,5 +140,17 @@ return network.registerProtocol('qmi', {
 			_('If unchecked, the advertised DNS server addresses are ignored'));
 		o.default = o.enabled;
 
+		o = s.taboption('advanced', form.Value, 'profile',
+			_('APN profile index'));
+		o.placeholder = '1';
+		o.datatype = 'uinteger';
+
+		if (L.hasSystemFeature('ipv6')) {
+			o = s.taboption('advanced', form.Value, 'v6profile',
+				_('IPv6 APN profile index'));
+			o.placeholder = '1';
+			o.datatype = 'uinteger';
+			o.depends('pdptype', 'ipv4v6');
```

Comment:
I expect the user to just use `profile` property in this case. The same stands for `apn`. The additional fields are to be used only for `ipv4v6` PDP type, otherwise they aren't needed.